src/radical/pilot/pilot_manager.py

```diff
@@ -426,40 +419,9 @@ def _pilot_staging_input(self, sds):
         sds  = expand_staging_directives(sds)
         uids = [sd['uid'] for sd in sds]
```

Comment:
`uids = ...` is not used later, could be removed

Reply:
Thanks, cleaned.